### PR TITLE
make dbfilter_from_header compliant with it's documentation

### PR DESCRIPTION
**Pull request by sbidoul**

### CODE REVIEW

File: dbfilter_from_header/__init__.py [hunk 1]
```diff
@@ -28,9 +28,7 @@
 def db_filter(dbs, httprequest=None):
     dbs = db_filter_org(dbs, httprequest)
     httprequest = httprequest or http.request.httprequest
-    db_filter_hdr = \
-        httprequest.environ.get('HTTP_X_ODOO_DBFILTER') or \
```
Comment (moylop260) [inline dbfilter_from_header/__init__.py:32]: What about use `HTTP_X_ODOO_DBFILTER` too?

**Comment by sbidoul** [comment]
@moylop260 I improved the patch.

**Comment by moylop260** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by guewen** [comment]
:+1:

**Pull request merged**